### DISCUSSION TITLE
Is this script also working on UXG Pro?

### DISCUSSION THREAD

**Discussion by Multied1** [fabianishere/udm-iptv#131] [Q&A]
Has someone tried this script on the UXG pro? Reason for asking, I'm using this script successfully for a couple of months now on the UDM Pro but not using any of the rest of the applications such as protect, talk ect and consider to buy a UXG pro.

Hope to hear from u and thanks in advance.

**Comment by fabianishere** [comment]
I think the UXG Pro uses an immutable root mount (like the UDM Pro), but I don’t think it runs a Debian container where you can install the package, since it does not have a controller.

I can’t verify it, since I don’t have the device myself, but I think it is safest to assume this packages does not work on it.

**Reply by Multied1** [reply]
Thanks for the reply Fabian, maybe then wait until someone can test it😉

**Reply by bigcityboyz1** [reply]
I have a UXG. Unfortunately, it fails to install the script. It gets stuck on:  chown: unknown user/group _apt:root

**Reply by LiviusNL** [reply]
Pull request #261 add support for UGX Pro. A
It requires an UGX pro with firmware 3.0.x
